homeassistant/components/freebox/button.py

```diff
@@ -64,7 +72,7 @@ def __init__(
         """Initialize a Freebox button."""
         self.entity_description = description
         self._router = router
-        self._attr_unique_id = f"{router.mac} {description.name}"
+        self._attr_unique_id = f"{router.mac} {description.key}"
```

Comment:
I have thought some more about this.
I think it is better to keep `.name` in this PR, and fix the `unique_id` in a separate PR, with a migration step.